### ISSUE TITLE
Recording Output is Silence when Record with Master Volume Muted/Disabled Except Your Own Microphone If You Talking At That Time

### ISSUE THREAD

**Issue by lbk2907** [BearWare/TeamTalk5#1665]
### Description

This only happend if you record using "single audio file for all users". It doesn't happend when you use "separate audio file for each user".
When you mute/disable your master volume by pressing Control+M, the recording also will got muted except your own microphone if you talking at that time. The recording also will follow the master volume. Let's say you just set your master volume to 20 percent and your microphone volume is set to 50 percent, your voice in that recording will louder than other people that are talking in that recording.
I consider this is a bug because certain people like me will use the separate client/profile just for record the conversation in the particular channel. In that case, they will mute the client/profile that they use for recording.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

The recording output should not follow the master volume state when record using "single audio file for all users".

### Actual behavior

The recording output is follow the master volume state when record using "single audio file for all users".

### Steps to reproduce problem

I forgot this bug happend start on which version of TeamTalk. What you need to do is just start the recording with "single audio file for all users". This still happend on the latest TeamTalk release.

**Comment by amirmahdifard** [comment]
@lbk2907 2 question to help the developers fix this more eezili: 1: do you have this problem in v5.11 too, since that vertion doesn't have overlapping and one by one modes: 2: wich playback mode are you using to get this bug? please try the default mode and see if your problem is fixed?

**Comment by lbk2907** [comment]
@amirmahdifard
This is about recording in TeamTalk. Anyway, to answer your question, yes. 
It happend on TeamTalk 5.11. The recording output is silenced when I mute 
my master volume.
Maybe you get confused with recording output. What I mean by recording 
output is the saved recording audio.

**Comment by amirmahdifard** [comment]
@lbk2907 yeah i know what is recording output, i just asked because  may be in 5.12 it happened to you due too new playback system: but if it is stil happening even in default mode, then this bug doesn't interduce in 5.12, it was there vefore

**Comment by lbk2907** [comment]
@amirmahdifard
Yes. It happend in the previous version also.

**Comment by amirmahdifard** [comment]
@lbk2907 good then, now we know that the problem is not from the teamtalk sound device playback: so lets wait until they fix it: i'm trying my best to report the Qt client problems so it will become really good as classic client.

**Comment by lbk2907** [comment]
@amirmahdifard
Thanks.

**Comment by amirmahdifard** [comment]
@lbk2907 you also if you know some programming, please have a look to problems and try opening pr and fixing some of them to inprove the qt client: thanks.
